tar/tarsnap.1-man.in

```diff
@@ -910,14 +914,29 @@ current position and exit (see "INTERRUPTING ARCHIVAL" below).
 .RE
 .SH PRINTING ARCHIVE STATISTICS
 .ad l
-Statistics on archives can be printed by running
-\fB\%tarsnap\fP
-\fB\--print-stats\fP
-and during archive creation or deletion statistics on the created or
-deleted archive can be printed using the
-\fB\--print-stats\fP
-option.
-In either case,
+There are three different commands which print statistics about archives:
+.RS 5
+.IP \(bu
+Global statistics:
+.RS 4
+\fB\%tarsnap\fP \fB\--print-stats\fP
+.RE
+.IP \(bu
+Global statistics and info about specific archive(s):
+.RS 4
+\fB\%tarsnap\fP \fB\--print-stats\fP \fB\-f\fP \fIarchive-name1\fP
+.RE
+.IP \(bu
```

Comment:
We should probably mention `--print-stats -f '*'` here?

Reply:
I thought that would be too much, but ok, added and rebased.  I also added `-f [...]` to the "info about specific archive(s)" line and deleting archives.